src/lib/utils/underscore-import-polyfill.js

```diff
@@ -26,7 +26,7 @@ window['_import'] = (src) => {
     const n = Object.assign(document.createElement('script'), {
       src: `/${src}`, // Rollup generates sources only in top-level
       type: 'module',
-      onload: () => resolve(),
+      onload: () => resolve(eval(`import('${n.src}');`)), // TODO: BAD!
```

Comment:
I honestly think we should now drop the dynamic `import()` polyfill now. This removes about 0.5% support level; mostly legacy Edge. When we started a year ago this number was bigger.

@robdodson WDYT? It would make our lives easier.

Reply:
@beaufortfrancois Can you remove all trace of the `_import` polyfill work?

Reply:
Sure. I'm ooo today but I'll take care of this tomorrow in a separate PR.

Reply:
@samthor Here it is: https://github.com/GoogleChrome/web.dev/pull/4211

Reply:
Please rebase and remove eval now that we use a real `import` :)

Reply:
@samthor Done.